buildsystem/DependencyFetch.cmake

```diff
@@ -0,0 +1,66 @@
+# Copyright 2017-2017 the openage authors. See copying.md for legal info.
+
+# cmake external project wrapper
+
+
+
+function(fetch_project)
+	set(KW_ARGS
+		NAME
+		LOCATION
+		TAG
```

Comment:
`LOCATION` and `TAG` seem unused (here and in the `ExternalFetch.cmake.in`). Can remove.
And `NAME` can be made the function's argument, removing the need to call `cmake_parse_arguments` below.